handler/src/test/java/io/netty/handler/ssl/SSLEngineTest.java

```diff
@@ -186,6 +193,205 @@ protected void channelRead0(ChannelHandlerContext ctx, ByteBuf msg) throws Excep
         }
     }
 
+    enum BufferType {
+        Direct,
+        Heap,
+        Mixed
+    }
+
+    @Parameterized.Parameters(name = "{index}: bufferType = {0}")
+    public static Collection<Object[]> data() {
```

Comment:
When there is only a single constructor parameter, you don't need to use `Object[]` but just Object.